tests/test_arrow.py

```diff
@@ -1157,6 +1157,58 @@ def test_bounds_param_is_passed(self):
             (arrow.Arrow(2013, 4, 1), arrow.Arrow(2013, 7, 1)),
         ]
 
+    def test_exact_bound_exclude(self):
+        result = list(
+            arrow.Arrow.span_range(
+                "hour",
+                datetime(2013, 5, 5, 12, 30),
+                datetime(2013, 5, 5, 17, 15),
+                exact=True,
+            )
+        )
+
+        assert result == [
```

Comment:
Mind storing the expected results in a variable called `expected` and then calling `assert result == expected`? It would make these tests look super clean :).

Reply:
Done! : )